generators/base/generator-base.mjs

```diff
@@ -346,25 +322,21 @@ export default class JHipsterBaseGenerator extends PrivateBase {
    * @param {string} packageJsonSourceFile - Package json filepath with actual versions.
    */
   replacePackageJsonVersions(keyToReplace, packageJsonSourceFile) {
```

Comment:
can be dropped

Reply:
Another problem of all those methods is easy to just forget about them.
The utilities created when split this method should be removed too.

Reply:
removed sir